From 2856aebfdf14b975ef30b6a5cc09a0dbb07eb7f0 Mon Sep 17 00:00:00 2001
From: Adam Duskett <Aduskett@gmail.com>
Date: Sat, 4 Apr 2020 23:18:36 -0700
Subject: [PATCH 01/10] make-compiler-variable-overwritable

Allow cross compilers such as buildroot or Yocto for specify a compiler.

Signed-off-by: Adam Duskett <Aduskett@gmail.com>
---
 gfx/video_filters/Makefile                 | 8 ++++----
 libretro-common/audio/dsp_filters/Makefile | 6 +++---
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/gfx/video_filters/Makefile b/gfx/video_filters/Makefile
index 7bd4a46709..4d36c8673b 100644
--- a/gfx/video_filters/Makefile
+++ b/gfx/video_filters/Makefile
@@ -1,10 +1,10 @@
-compiler    := gcc
-extra_flags :=
-use_neon    := 0
+compiler    ?= gcc
+extra_flags ?=
+use_neon    ?= 0
 release	    := release
 DYLIB	    := so
 PREFIX      := /usr
-INSTALLDIR  := $(PREFIX)/lib/retroarch/filters/video
+INSTALLDIR  ?= $(PREFIX)/lib/retroarch/filters/video
 
 ifeq ($(platform),)
    platform = unix
diff --git a/libretro-common/audio/dsp_filters/Makefile b/libretro-common/audio/dsp_filters/Makefile
index 34392e173c..2384273e4e 100644
--- a/libretro-common/audio/dsp_filters/Makefile
+++ b/libretro-common/audio/dsp_filters/Makefile
@@ -1,10 +1,10 @@
-compiler    := gcc
+compiler    ?= gcc
 extra_flags :=
 use_neon    := 0
 build       = release
 DYLIB	      := so
-PREFIX      := /usr
-INSTALLDIR  := $(PREFIX)/lib/retroarch/filters/audio
+PREFIX      ?= /usr
+INSTALLDIR ?= $(PREFIX)/lib/retroarch/filters/audio
 
 ifeq ($(platform),)
    platform = unix
-- 
2.34.1

